project_euler/problem_27/problem_27_sol1.py

```diff
@@ -1,4 +1,9 @@
 """
+Euler Problem 27
```

Comment:
```suggestion
Project Euler Problem 27
```
Also, please change the filename to just `sol1.py`